alphastats/gui/pages/02_Import Data.py

```diff
@@ -1,76 +1,141 @@
 import streamlit as st
-import os
-import io
-
-try:
-    from alphastats.gui.utils.import_helper import (
-        import_data,
-        save_plot_sampledistribution_rawdata,
-        display_loaded_dataset,
-        load_sample_data,
-        empty_session_state,
-    )
-    from alphastats.gui.utils.ui_helper import sidebar_info
-
-except ModuleNotFoundError:
-    from utils.ui_helper import sidebar_info
-    from utils.import_helper import (
-        import_data,
-        save_plot_sampledistribution_rawdata,
-        display_loaded_dataset,
-        load_sample_data,
-        empty_session_state,
-    )
 
-from streamlit.runtime import get_instance
-from streamlit.runtime.scriptrunner.script_run_context import get_script_run_ctx
+from alphastats import DataSet
+from alphastats.gui.utils.options import SOFTWARE_OPTIONS
+
+from alphastats.gui.utils.import_helper import (
+    save_plot_sampledistribution_rawdata,
+    load_example_data,
+    empty_session_state,
+    load_softwarefile_df,
+    show_metadata_file_uploader,
+    show_loader_columns_selection,
+    load_proteomics_data,
+    load_options,
+    show_select_sample_column_for_metadata,
+    init_session_state,
+)
+from alphastats.gui.utils.ui_helper import sidebar_info
+
+init_session_state()
 
-runtime = get_instance()
-session_id = get_script_run_ctx().session_id
-# session_info = runtime._session_mgr.get_session_info(session_id)
-# TODO: remove this line at some point if we really don't need it
+sidebar_info()
 
-user_session_id = session_id
-st.session_state["user_session_id"] = user_session_id
+st.markdown("### Start a new session")
+st.write(
+    "Start a new session will discard the current one (including all analysis!) and enable importing a new dataset."
+)
+st.write("To explore AlphaPeptStats you may also load an example dataset.")
 
-if "loader" not in st.session_state:
-    st.session_state["loader"] = None
+c1, c2 = st.columns(2)
+if c1.button("Start new Session"):
+    empty_session_state()
+    st.rerun()
 
-if "gene_to_prot_id" not in st.session_state:
-    st.session_state["gene_to_prot_id"] = {}
+if c2.button("Start new Session with example DataSet"):
+    empty_session_state()
+    init_session_state()
+    loader, metadata_columns, dataset = load_example_data()
 
-if "organism" not in st.session_state:
-    st.session_state["organism"] = 9606  # human
+    st.session_state["dataset"] = dataset
+    st.session_state["metadata_columns"] = metadata_columns
+    st.session_state["loader"] = loader
+    load_options()
+    # TODO why are we doing this so early?
+    save_plot_sampledistribution_rawdata(dataset)
+    sidebar_info()
+    st.stop()
 
-sidebar_info()
 
-if "dataset" not in st.session_state:
-    st.markdown("### Import Proteomics Data")
+st.markdown("### Import Proteomics Data")
+if "dataset" in st.session_state:
+    st.info(f"DataSet already present: {st.session_state['dataset']}")
```

Comment:
It would be great to provide additional info here and maybe show the preview. Also maybe refer to the sidebar and the data overview.

Reply:
I would not double UI elements for now, will add a link to the "data overview" page